src/main/java/org/troyargonauts/subsystems/DriveTrain.java

```diff
@@ -102,15 +103,15 @@ public void tankDrive(double left, double right, double nerf) {
      * @return encoder position based on encoder values
      */
     public double getPosition() {
-        return (frontRight.getEncoder().getPosition() + frontLeft.getEncoder().getPosition()) / (2 * DriveConstants.GEARBOX_SCALE);
+        return (frontRight.getEncoder().getPosition() + frontLeft.getEncoder().getPosition()) / (2 * Constants.DriveTrain.GEARBOX_SCALE);
     }
 
     public double getLeftPosition() {
-        return frontLeft.getEncoder().getPosition() / DriveConstants.GEARBOX_SCALE;
+        return frontLeft.getEncoder().getPosition() / Constants.DriveTrain.GEARBOX_SCALE;
     }
 
     public double getRightPosition() {
```

Comment:
Add javadocs, and return the average of the variables set within periodic. Remove Gearbox scale, we want the actual averaged encoder values to compare if any issues arise.